libnymea-core/ruleengine.cpp

```diff
@@ -1217,6 +1281,7 @@ void RuleEngine::appendRule(const Rule &rule)
 {
     Rule newRule = rule;
     newRule.setStatesActive(newRule.stateEvaluator().evaluate());
+    qCDebug(dcRuleEngineDebug()) << "Adding Rule:" << newRule;
```

Comment:
Maybe we should print this with `dcRuleEngine`, since this is always interesting? And it does not happen that often, but always good to read.